redbot/cogs/audio/core/utilities/player.py

```diff
@@ -65,6 +65,23 @@ async def get_active_player_count(self) -> Tuple[Optional[str], int]:
             playing_servers = 0
         return get_single_title, playing_servers
 
+    async def get_current_track_title(self) -> str:
```

Comment:
Since this method reuses code from `get_active_player_count()`, we should update `get_active_player_count()` to just call it.